test/addLinkToProjectResponse.test.js

```diff
@@ -0,0 +1,18 @@
+const { expect } = require('chai');
+
+const AddLinkToProjectResponse = require('../lib/actions/addLinkToProjectResponse');
+
+describe('Add Link To Project Response Mock', () => {
+  it('returns true if bot response link is stored', () => {
+    const expectedMessage = 'Link stored.'
+    let hasReplied = false;
+    function replyMock(message) {
+      hasReplied = message === expectedMessage;
+    }
+    
+    const response = new AddLinkToProjectResponse();
+    response.sendToBot( {reply: replyMock} );
```

Comment:
`sendToBot` is async. 

1. does it need to be?
2. don't you need to wait on that to finish before doing your assertion? Are you just getting luck that it finishes fast enough?

Reply:
so this test should be async as well. I do believe that send to bot needs to be async because it's tied to to an async method (execute). but maybe I'm thinking about that in the wrong way.. maybe it doesn't necessarily need to be.

Reply:
`execute` is async, but this is the return value of `execute`. You don't need this to be async.

Reply:
got it!